src/blocks/author-list/components/AdvancedStyling.js

```
@@ -0,0 +1,164 @@
+					label={ __( 'Saturation', 'author-profile-blocks' ) }
+					value={ filterSaturate }
+					onChange={ ( value ) => onChange( { filterSaturate: value } ) }
+					min={ 0 }
+					max={ 200 }
+					step={ 5 }
+					help={ __( 'Adjust saturation (100 = normal)', 'author-profile-blocks' ) }
+				/>
+			</div>
+		</PanelBody>
+	);
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)